docs/get-started/sumo-logic-ui-new.md

```diff
@@ -0,0 +1,215 @@
+
+If you're an admin, you can view the Library in [Content Administrator](#search-content-administrator-library) mode.
+
+:::tip
+Enlarge your working area by hiding the left-side nav panel. Just click the hamburger menu icon. To unhide it, click the hamburger menu icon again.<br/><img src={useBaseUrl('/img/get-started/hamburger.gif')} alt="hamburger.gif" width="350"/>
+:::
+
+## Using the top navigation bar
+
+### Access Apps, Admin settings
+
+The global toolbar (top nav bar) provides you access to (from left to right): [Help](#get-help-docs-community-and-more), [App Catalog](apps-integrations.md), [(Data) Configuration](/docs/send-data), [Administration](/docs/manage), and your user profile options ([Notifications](account-settings-preferences.md) and [Preferences](account-settings-preferences.md)).
+
+<img src={useBaseUrl('/img/get-started/admin-config.png')} alt="admin-config.png" width="350"/>
+
+### Manage your personal account preferences
+
+You can manage your personal account settings from the **Preferences** page. These settings apply only to your account. Changes you make to your preferences take effect the next time you sign in, not during the current session.
+
+To manage your personal Sumo account preferences:
+
+1. From the top nav bar, click the person icon, and then from the dropdown, select **Preferences**.<br/><img src={useBaseUrl('img/get-started/acct-pref.png')} alt="img/get-started/acct-pref.png" width="350"/>
+1. In the Preferences page, you can modify the following settings:
+    * **My Profile**. Username and password.
+    * **My Access Keys**. Add, edit, and remove access keys.
+    * **My Preferences**. Your account session settings.
+
+For more information, see [Account Preferences and Credentials](account-settings-preferences.md).
+
+## Customize your environment with tabs
+
+If you'd prefer to multitask and keep multiple tab open simultaneously (for example: log search, dashboards, App Catalog, and Preferences), we recommend utilizing your own web browser's tab grouping functionality. By adding Sumo Logic tabs to a tab group, any new tabs opened within the Sumo Logic platform will automatically open in the same tab group.
+
+<img src={useBaseUrl('/img/get-started/tabs.png')} alt="tabs.png" width="500" />
+
+This will also allow you to collapse the tab group to reclaim valuable real estate in your browser's tab bar.
+
+## Mastering everyday tasks
+
+This section provides information on how to perform basic everyday tasks using the Sumo Logic UI.
+
+### Analysts (all users)
+
+* [Launch log searches, metrics visualizations, and Live Tail sessions](#launch-searches-metrics-visualizations-and-live-tail-sessions)
+* [View recent dashboards and searches](#view-recent-dashboards-and-searches)
+* [View Favorites and add dashboards and searches to the list](#view-favorites-and-add-dashboards-and-searches-to-the-list)
```

Comment:
The link on "View Favorites and add dashboards and searches to the list" is broken because there is no corresponding section to link to.